.github/workflows/main.yml

```diff
@@ -1,11 +1,12 @@
 # yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
 
 name: publish
+
 on:
   workflow_dispatch: # Allow running the workflow manually from the GitHub UI
   push:
     branches:
-      - 'release/*'  # Run the workflow when pushing to release branches like release/{VersionNumber}
+      - 'main'       # Run the workflow when pushing to the main branch
```

Comment:
Do we want to add the develop branch to continue deploying beta releases?

Reply:
The develop branch **can** still deploy a beta release. We just need to create a release on github with the target branch as develop
The above code just starts a pipeline build whenever there are pushes to the ```main``` branch, or any PRs
Beta releases are only deployed when an actual release is created in github with a certain tag name syntax